index.md

```diff
@@ -74,21 +69,22 @@ impose a common storage format that all repository systems need to implement.
 While it can be used as an archival storage format, it can also be seen as a
 format that makes system migration easier.
 
-# Preliminary definitions and remarks
+# Definitions and remarks
 
 <a name="ip"></a>
 
-## Information package (IP)
+## Information package
+
+The CSIP defines the requirements for concrete information package format
```

Comment:
The CSIP defines the SHARED? requirements

Reply:
"shared" added